cinnabar/plotting.py

```diff
@@ -15,7 +15,7 @@ def _master_plot(
     yerr: Optional[np.ndarray] = None,
     method_name: str = "",
     target_name: str = "",
-    quantity: str = r"$\Delta \Delta$ G",
+    quantity: str = r"$\Delta$ $\Delta$ G",
```

Comment:
We sure about this? I feel like the previous used to work, so just wanted to double check we checked this.